package.json

```diff
@@ -9,6 +9,8 @@
     "@emotion/styled": "^11.6.0",
     "@fullcalendar/daygrid": "^5.10.1",
     "@fullcalendar/react": "^5.10.1",
+    "@material-ui/core": "^4.12.3",
```

Comment:
Not sure we should commit these... These package files didn't work on my machine.